### DISCUSSION TITLE
Download source code to run locally

### DISCUSSION THREAD

**Discussion by triumphhub** [nhn/tui.calendar#1343] [Q&A] [answered]
Is there a link to download source code to run locally. I've looked in several of the download packages and links appear to be broken in the example html files which refer to a "../dist" folder for css and js files. Where do I find the "dist" folder with css and js files?

Thanks for an awesome job and any help

**Comment by adhrinae** [comment, marked as answer]
you need to clone this repository and build the package to run examples

https://github.com/nhn/tui.calendar/issues/1317#issuecomment-1325796560

**Reply by triumphhub** [reply]
Thank you I don't know what "build the package" means so I'll do some more research.

**Reply by adhrinae** [reply]
If you're not familiar with the front-end ecosystem, that might be challenging.

If you simply wish to run examples on your machine, try modifying the `../dist` path in the HTML files mentioned below:

```
https://unpkg.com/@toast-ui/calendar@2.1.3/dist
```

**Reply by triumphhub** [reply]
I appreciate your time as this is probably way off topic. I installed npm on a Linux server and downloaded the package using "npm install --save @toast-ui/calendar". I see there are many "dist" subfolders that I'm going through manually. Is there a map document of where stuff is? 

Thanks once again.

**Reply by adhrinae** [reply]
Let's start over.

I assume you'd cloned the repository to run examples in the `apps/calendar/examples` folder locally but you couldn't because there is no `../dist` folder in HTML files.

https://github.com/nhn/tui.calendar/blob/main/apps/calendar/examples/00-calendar-app.html#L124

**When you build the package, the bundle files will be created into the `apps/calendar/dist` folder. that is what the `dist` folder means.** But you don't understand how to build the package.

At least you have installed Node.js(v16+ is recommended) on your server so try following the steps below:

1. Clone the repository
2. Run the `npm install` command in the project root folder.
3. Go to the `apps/calendar` folder.
4. Run the `npm run build` command.
5. You can see the `dist` folder in the `apps/calendar` folder.

**Reply by triumphhub** [reply]
Thank you for all your help, I am extremely grateful. I've followed your instructions (and a few tutorials) and have found everything I need.
